tests/Extension/ItemRestrictionsTest.php

```diff
@@ -267,8 +268,8 @@ public function testIsWithNot()
 
     public function testFollowedBy()
     {
-        //TODO:
-        return 0;
+        $this->markTestSkipped('TODO');
```

Comment:
Had to add this, otherwise phpunit6 reports this test case as error, because no assertions are being performed.